tests/test_cloze.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+"""Test creating Cloze cards"""
+
+from genanki import Model
+from genanki import Note
+from genanki import Deck
+from genanki import Package
+
+CSS = """.card {
+ font-family: arial;
+ font-size: 20px;
+ text-align: center;
+ color: black;
+ background-color: white;
+}
+
+.cloze {
+ font-weight: bold;
+ color: blue;
+}
+.nightMode .cloze {
+ color: lightblue;
+}
+"""
+
+MY_CLOZE_MODEL = Model(
+  998877661,
+  'My Cloze Model',
+  fields=[
+    {'name': 'Text'},
+    {'name': 'Extra'},
+  ],
+  templates=[{
+    'name': 'My Cloze Card',
+    'qfmt': '{{cloze:Text}}',
+    'afmt': '{{cloze:Text}}<br>{{Extra}}',
+  },],
+  css=CSS,
+  model_type=1)
+
+def test_cloze():
+  """Test Cloze model"""
+  notes = []
+  assert MY_CLOZE_MODEL.to_json(0, 0)["type"] == 1
+
+  fields = ['NOTE ONE: {{c1::single deletion}}', '']
+  my_cloze_note = Note(model=MY_CLOZE_MODEL, fields=fields)
+  assert {card.ord for card in my_cloze_note.cards} == {0}
+  notes.append(my_cloze_note)
+
+  fields = ['NOTE TWO: {{c1::1st deletion}} {{c2::2nd deletion}} {{c3::3rd deletion}}', '']
+  my_cloze_note = Note(model=MY_CLOZE_MODEL, fields=fields)
+  assert {card.ord for card in my_cloze_note.cards} == {0, 1, 2}
+  notes.append(my_cloze_note)
+
+  fields = ['NOTE THREE: {{c1::1st deletion::C1-CLOZE}}', '']
```

Comment:
Could you add another test case where there are multiple Cloze deletions with the same index, e.g.
```
'NOTE FOUR: {{c1:1st deletion}} foo {{c2:2nd deletion}} bar {{c1::3rd deletion}}'
```
This is documented [in the manual](https://apps.ankiweb.net/docs/manual20.html#cloze-deletion):
> You can also elide multiple sections on the same card. In the above example, if you change c2 to c1, only one card would be created, with both Canberra and 1913 hidden.

Reply:
Good catch. I am working on it...